### ISSUE TITLE
As a screen reader user, I cannot find the screen to edit load balancers

### ISSUE THREAD

**Issue by ndarilek** [rancher/rancher#3094]
In the previous UI, I was able to edit load balancers by tabbing to the menu, pressing enter, then arrowing to the option. Now, while I can find an option to edit the specific container, I can no longer tab to the option that allows manipulating the load balancer service as a whole. I can't find this option at all with my screen reader, actually.

Can someone please ensure that this option is in the tab order and that it can be used via the keyboard? I have a Rancher deployment I can no longer administer as a result of this regression, and I can't seem to get upgrading to work by rancher-compose so command line use is out for the moment.

Also, as an aside, please use aria-label attributes to assign text labels to some of your icon links. I also can't seem to find functionality for exporting a zip of my configs, though I did eventually get a text dump of the YAML by clicking on unlabeled icons at random. Needless to say, I think that clicking randomly on icons on a tool managing a live deployment is a bad idea, but as a blind user it's all you've left me with. :)

Thanks.

**Comment by ndarilek** [comment]
As an aside, I've since found this menu, but you're still missing `aria-label` attributes for the `<button/>` elements that as of now only have an icon.

See also #3674

**Comment by tfiduccia** [comment]
Version - master 3/22
Verified fixed
